src/bundle/Resources/public/scss/_adaptive-filters.scss

```diff
@@ -81,19 +81,19 @@
         align-items: flex-end;
         height: calculateRem(79px);
         overflow: hidden;
+
+        &--after-initialization {
+            overflow: visible;
+        }
     }
 
     &__collapsible {
         border-top: calculateRem(1px) solid $ibexa-color-light;
-        transition: all $ibexa-admin-transition-duration $ibexa-admin-transition;
+        transition: none;
```

Comment:
do we need `none`, are we overriding something?

Reply:
Even though it does not override other `transition` it affects animation.